### PR TITLE
Add basic PyTorch DALI example, fix links to files in docs

### PR DESCRIPTION
**Pull request by JanuszL**
- adds very basic example how to use DALI in PyTorch
- fixes links in the docs to the GitHub and adds few more of them

Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

### CODE REVIEW

File: docs/examples/tensorflow/demo/nvutils/image_processing.py [hunk 1]
```diff
@@ -98,8 +98,8 @@ def __init__(self,
             output_dtype=dali.types.FLOAT,
             crop=(height, width),
             image_type=dali.types.RGB,
-            mean=[121., 115., 100.],
-            std=[70., 68., 71.],
+            mean=[0.485 * 255,0.456 * 255,0.406 * 255],
+            std=[0.229 * 255,0.224 * 255,0.225 * 255]),
```
Comment (awolant) [inline docs/examples/tensorflow/demo/nvutils/image_processing.py]: Could you explain why this happend? I thought that these constants were specific to PyTorch pretrained models

Comment (JanuszL) [reply docs/examples/tensorflow/demo/nvutils/image_processing.py]: Leftover from other things, reverting.

File: docs/examples/pytorch/pytorch-basic_example.ipynb [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Using DALI in PyTorch\n",
+    "\n",
+    "### Overview\n",
+    "\n",
+    "This example shows how to use DALI in pytorch.\n",
+    "\n",
+    "The CaffeReader reader is used in this example.\n",
```
Comment (mzient) [inline docs/examples/pytorch/pytorch-basic_example.ipynb]:
```suggestion
    "This example uses CaffeReader.\n",
```

File: docs/examples/pytorch/pytorch-basic_example.ipynb [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+    "For details on how to use other data type please see other [examples](..)."
```
Comment (mzient) [inline docs/examples/pytorch/pytorch-basic_example.ipynb]:
```suggestion
    "See other [examples](...) for details on how to use different data formats."
```

File: docs/examples/pytorch/pytorch-basic_example.ipynb [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Using DALI in PyTorch\n",
+    "\n",
+    "### Overview\n",
+    "\n",
+    "This example shows how to use DALI in pytorch.\n",
```
Comment (mzient) [inline docs/examples/pytorch/pytorch-basic_example.ipynb]:
```suggestion
    "This example shows how to use DALI in PyTorch.\n",
```

Comment (Kh4L) [inline docs/examples/pytorch/pytorch-basic_example.ipynb]:
```suggestion
    "This example shows how to use DALI in PyTorch.\n",
```

File: docs/examples/pytorch/pytorch-basic_example.ipynb (hunk 3 above)
Comment (mzient) [reply docs/examples/pytorch/pytorch-basic_example.ipynb]: ...or maybe move these comments (see below) - your call.

File: docs/examples/pytorch/pytorch-basic_example.ipynb [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+    "BATCH_SIZE = 128  # batch size per GPU\n",
+    "ITERATIONS = 32\n",
+    "IMAGE_SIZE = 3"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let us define pipeline with the reader:"
```
Comment (mzient) [inline docs/examples/pytorch/pytorch-basic_example.ipynb]:
```suggestion
    "Let us define a pipeline with a reader:"
```
Additionally, the comment about using CaffeReader and refering to other examples could be moved here.

File: docs/examples/pytorch/pytorch-basic_example.ipynb [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+    "        output = self.cmn(images, crop_pos_x = self.uniform(),\n",
+    "                          crop_pos_y = self.uniform())\n",
+    "        return (output, labels)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let us create pipeline and pass it to PyTorch generic iterator"
```
Comment (mzient) [inline docs/examples/pytorch/pytorch-basic_example.ipynb]:
```suggestion
    "Let us create the pipeline and pass it to PyTorch generic iterator"
```

File: docs/examples/pytorch/pytorch-basic_example.ipynb (hunk 4 above)
Comment (JanuszL) [reply docs/examples/pytorch/pytorch-basic_example.ipynb]: Done

File: docs/examples/pytorch/pytorch-basic_example.ipynb (hunk 6 above)
Comment (JanuszL) [reply docs/examples/pytorch/pytorch-basic_example.ipynb]: Done

File: docs/examples/pytorch/pytorch-basic_example.ipynb (hunk 5 above)
Comment (JanuszL) [reply docs/examples/pytorch/pytorch-basic_example.ipynb]: Done

File: docs/examples/pytorch/pytorch-basic_example.ipynb (hunk 4 above)
Comment (JanuszL) [reply docs/examples/pytorch/pytorch-basic_example.ipynb]: Done

File: docs/examples/pytorch/pytorch-basic_example.ipynb (hunk 2 above)
Comment (JanuszL) [reply docs/examples/pytorch/pytorch-basic_example.ipynb]: Done

File: docs/examples/pytorch/pytorch-basic_example.ipynb (hunk 3 above)
Comment (JanuszL) [reply docs/examples/pytorch/pytorch-basic_example.ipynb]: Done

**Pull request merged**